api/app/schemas/observations.py

```diff
@@ -19,6 +19,8 @@ class WeatherReading(BaseModel):
     ffmc: float = None
     isi: float = None
     fwi: float = None
+    observation_valid: bool = None
```

Comment:
Ooooh - can you give this a try? Don't know if it will work (you have to add from typing import Union) - but if that works, then we will get rid of Sonarcloud complaining (if it works, can you do it for all the other values?) 
```suggestion
    observation_valid: Union(bool, None) = None
```